code/modules/projectiles/ammo_datums.dm

```diff
@@ -989,6 +989,20 @@ datum/ammo/bullet/revolver/tp44
 	accurate_range_min = 2
 	damage_falloff = 0.25
 
+/datum/ammo/bullet/sniper/pfc/flak
+	name = "high caliber flak rifle bullet"
+	hud_state = "sniper_supersonic"
+	flags_ammo_behavior = AMMO_BALLISTIC|AMMO_SUNDERING|AMMO_SNIPER
+	damage = 70
```

Comment:
Make it half damage, and half penetration from the normal rounds. Staggerstun is pretty powerful. Halve em and it looks good to me

Reply:
Done!